README.md

```diff
@@ -1,43 +1,30 @@
-# The One and Only
+# merge-conflict-demo
```

Comment:
I preferred the old title.


Perhaps with an updated twist?
```suggestion
# The One and Only Matt
```

Reply:
sweet